ui/app/src/modules/Search/FilterSearchDropdown.tsx

```diff
@@ -165,6 +165,14 @@ const messages: any = defineMessages({
     id: 'searchFilter.desc',
     defaultMessage: 'Descending'
   },
+  moreRelevance: {
```

Comment:
moreRelevantFirst: {
 id: searchFilter.byRelevanceDescMessage
 default...
},
lessRelevantFirst: {
 id: searchFilter.byRelevanceAscMessage
 default...
}

Reply:
Changes not complete (message id)

Reply:
updated

Reply:
Not updated: message descriptor ids are still not as requested.